docs/ref/models/fields.txt

```diff
@@ -94,38 +94,86 @@ and the second element is the human-readable name. For example::
         ('SO', 'Sophomore'),
         ('JR', 'Junior'),
         ('SR', 'Senior'),
+        ('GR', 'Graduate'),
     ]
 
-Generally, it's best to define choices inside a model class, and to
-define a suitably-named constant for each value::
+Django provides special enumeration types you can subclass to define this
+iterable in a concise way::
```

Comment:
> Generally, it's best to define choices inside a model class, and to
define a suitably-named constant for each value

I think this particular point has been lost, and is important. 

Later you `Card.Suit` being declared as a nested class, but it's only implicit what's going on. 

I **think** I'd leave the `YEAR_IN_SCHOOL_CHOICES` examples as are, through to the `self.JUNIOR` usage example, and then introduce Enum approach. (But not hung up on that particular ordering...)

Reply:
Thanks for the review. I restored this section and then introduced the enum approach as suggested.